crates/cxx-qt/src/lib.rs

```diff
@@ -357,3 +359,36 @@ where
         Self::initialize(self);
     }
 }
+
+/// Write the cxx-qt headers to the specified directory.
+pub fn write_headers(directory: impl AsRef<Path>) {
```

Comment:
would we want this documented?

Reply:
do we instead want a `cxx-qt-headers` as this then means we only have to list `cxxqt_connection` once

Reply:
done